internal/mocks/atlas/provider.go

```diff
@@ -3,6 +3,8 @@ package atlas
 import (
 	"context"
 
+	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/atlas"
```

Comment:
Please put this one with the other import  from `github.com/mongodb/mongodb-atlas-kubernetes/v2` so that goimports can order it properly.

Canonical imports structure we would like to converge to is:
- Stdlib packages at the top.
- External dependencies in the middle.
- Internal dependencies within AKO at the bottom.

Reply:
This is done with the goimport and linted with the golangci lint. I didn't put it there manually.

Reply:
This happens to me all the time, you slip an extra end of line, create a new group. Then `goimports` respects all the divisions you do. If you have 5 divisions it will sort each of the 5 divisions. But we only want to have 3. @s-urbaniak wanted to start using https://github.com/daixiang0/gci so that there is no ambiguity on our imports ordering and layout.

Reply:
agreed with @josvazg  but not a blocker for this PR